.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
     TASKS="bench:compile"
   - TEST_DIR=offline-regression-report
     TASKS="bench:compile"
+  - TEST_DIR=invocation-count-measurers
+    TASKS="bench:compile"
```

Comment:
Was thinking about the same. Do you prefer to do it in a separate PR or in this one?